front-end/src/components/main-page/TranscriptDescription.jsx

```diff
@@ -10,6 +10,7 @@ class TranscriptDescription extends Component {
                 </ul>
                 <pre className='overflow-x-scroll leading-7 text-sm bg-off-white shadow-inner-lg rounded p-2 my-4 mx-2'
                      data-testid="sampleTranscript"
+                     tabIndex = "0"
```

Comment:
Unneeded

Reply:
Are you sure? For this one you need to be able to scroll

Reply:
I believe a screen reader could handle it? Not 100% sure.

Reply:
I'm honestly not sure. But I feel like there might be people just using the keyboard but without screen reader?

Reply:
The main purpose of tabbing is for interaction not screen reading

Reply:
Ok I don't have enough information about this. So being able to scroll doesn't count as interaction?

Reply:
I think so (emphasis on think)

Reply:
https://accessibilityinsights.io/info-examples/web/scrollable-region-focusable/
I'm not sure how much we can trust this website 🤔

Reply:
That seems like strong insight. Definitely right that it is needed!